src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java

```diff
@@ -366,6 +366,12 @@
     )
     private YesOrNo applicant1NotifiedCanApplyForConditionalOrder;
 
+    @CCD(
+        label = "Notification sent to Applicant indicating that the dispute form is overdue",
+        access = {DefaultAccess.class}
+    )
+    private YesOrNo applicantNotifiedDisputeFormOverdue;
```

Comment:
Shouldn't this be in AcknowledgementOfService

Reply:
yes, agreed. it does seem like it belongs there. I'll move it

Reply:
done